test/test_van/test_email.py

```diff
@@ -102,10 +152,33 @@ def test_get_email_messages(self, m):
     def test_get_email_message(self, m):
         m.get(
             self.van.connection.uri + "email/message/1",
-            json=mock_response[0],
+            json=mock_response_enriched[0],
```

Comment:
Should be enriched here, because we call `get_email` with the default `expand` kwarg (`= True`). We could potentially add a test if we want `expand=False`, but I don't know if it's worth the effort.

Reply:
NVM, decided it was easy enough to add.